opm/grid/cpgrid/CpGridData.hpp

```diff
@@ -132,12 +137,12 @@ class CpGridData
     /// Constructor
     /// \param grid  The grid that we are the data of.
     explicit CpGridData(CpGrid& grid);
-#if HAVE_MPI
+
     /// Constructor for parallel grid data.
     /// \param comm The MPI communicator
     /// Default constructor.
-    CpGridData(MPI_Comm comm);
-#endif
+    explicit CpGridData(MPIHelper::MPICommunicator comm);
```

Comment:
copy intended? (e.g. it could lead to a MPI_Comm_dup in the wrapper class?)

Reply:
Just double checked. `MPIHelper::MPICommunicator` is `MPI_Comm`. There will no `MPI_Comm_dup`. Yes intended.